src/routes/api/user.routes.js

```diff
@@ -159,6 +161,8 @@ router.get('/google/callback', passport.authenticate('google', {
 router.post('/signup', checkFirstName, checkLastName, checkValidEmail, checkExistingEmail, checkPassword, validateResult, userController.signUp);
 router.get('/verify/:token', userController.verifyAccount);
 
+router.post('/assignRoles', checkValidEmail, checkRoles, validateResult, checkUser, userController.assignRole);
```

Comment:
```suggestion
router.post('/assign-roles', checkValidEmail, checkRoles, validateResult, checkUser, userController.assignRole);
```
Would you please adhere [restful standards](https://restfulapi.net/resource-naming/) for naming endpoints?

Reply:
Thanks, @hakoemmy I resolved it.